Jenkinsfile

```diff
@@ -71,13 +71,13 @@ pipeline {
                                 }
                             }
                         }
-                        stage('Check documentation') {
-                            steps {
-                                dir('mhs/outbound') {
-                                    sh label: 'Check API docs can be generated', script: 'pipenv run generate-openapi-docs > /dev/null'
-                                }
-                            }
-                        }
+//                         stage('Check documentation') {
```

Comment:
why is that commented out?

Reply:
Matt Lewis asked me to disable generating of the open api. Shouldn't it be commented out, @grzegorz-robak?

Reply:
I don't know why he asked you for it. If it isn't needed anymore then we should delete it. Or if it should be commented out for some time then reason for this should be placed as comment also.

Reply:
Matt Lewis confirmed, that it should be deleted. I understood it wrong before, I thought it is jus temporary.
So I am removing it

Reply:
I asked because we had a lot of queries in regards to the API spec from our test partner.  @petervdm and myself discussed this and agreed that the current auto generator should stop being used for MHS and all future adaptors